sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -653,3 +654,29 @@ def upgrade_scylla_cluster(self, new_version: str) -> None:
     @staticmethod
     def fstrim_scylla_disks():
         LOGGER.warning("Local K8S backends don't support running 'fstrim' command. Ignoring.")
+
+    def prefill_cluster(self, dataset_name: str):
+        test_data = getattr(datasets, dataset_name, None)
+        if not test_data:
+            raise ValueError(f"Dataset unexpected value: '{dataset_name}'. "
+                             f"Dataset with this name does not defined in the sdcm.utils.sstable.load_inventory")
+
+        test_keyspace_name = 'keyspace1'
+
+        replication_factor = len(self.nodes)
```

Comment:
@vponomaryov all the functional test has 3 nodes, right ? non of them start with less then 3 ?

Reply:
Right now nothing reads them, but if some of the tests would start reading or writing using cassandra-stress, we should have correct value, also I believe it would affect where how many nodes would participate in streaming while remove/adding nodes. 

So I think it's o.k. to hardcode it to 3

Reply:
changed to hardcoded = 3